src/maker/config.rs

```diff
@@ -29,23 +33,251 @@ pub struct MakerConfig {
     pub min_size: u64,
 }
 
+impl Default for MakerConfig {
+    fn default() -> Self {
+        Self {
+            port: 3000,
+            heart_beat_interval_secs: 3,
+            rpc_ping_interval_secs: 60,
+            watchtower_ping_interval_secs: 300,
+            directory_servers_refresh_interval_secs: 60 * 60 * 12, //12 Hours
+            idle_connection_timeout: 300,
+            onion_addrs: "onion@example".to_string(),
+            absolute_fee_sats: 1000,
+            amount_relative_fee_ppb: 10_000_000,
+            time_relative_fee_ppb: 100_000,
+            required_confirms: 1,
+            min_contract_reaction_time: 48,
+            min_size: 10_000,
+        }
+    }
+}
+
 impl MakerConfig {
     /// Init a default configuration with given port and address
-    pub fn init(port: u16, onion_addrs: String) -> Self {
-        Self {
+    pub fn init(port: u16, onion_addrs: String, file_path: Option<&PathBuf>) -> Self {
```

Comment:
No port and onion please in the args please. We can hardcode the default port to a specific number. 6102 sounds good to you?

If user specify a port in the config, then use that.

Hardcode the default onion to a random value for now. No need for it to be a config value.

Reply:
ok cool!!! Let me quickly add that...